docs/querying/groupbyquery.md

```diff
@@ -172,8 +175,8 @@ The subtotals feature allows computation of multiple sub-groupings in a single q
 }
 ```
 
-Response returned would be equivalent to concatenating result of 3 groupBy queries with "dimensions" field being ["D1", "D2", D3"], ["D1", "D3"] and ["D3"] with appropriate `DimensionSpec` json blob as used in above query.
-Response for above query would look something like below...
+The result of the subtotalSpec would be equivalent to concatenating the result of three groupBy queries, with the "dimensions" field being `["D1", "D2", D3"]`, `["D1", "D3"]` and `["D3"]`, given the `DimensionSpec` shown above.
+The response for the query above query would look something like: 
```

Comment:
The query in `the query above query` seems duplicate?

Reply:
yep, fixed